dashboard/src/modules/components/ToastNotificationComponent/index.jsx

```diff
@@ -18,27 +18,26 @@ const ToastComponent = () => {
   };
   return (
     <AlertGroup isToast>
-      {alerts &&
-        alerts.map((item) => {
-          return (
-            <Alert
-              variant={AlertVariant[item.variant]}
-              title={item.title}
-              key={item.key}
-              timeout={true}
-              onTimeout={() => removeToast(item.key)}
-              actionClose={
-                <AlertActionCloseButton
-                  title={item.title}
-                  variantLabel={`${item.variant} alert`}
-                  onClose={() => removeToast(item.key)}
-                />
-              }
-            >
-              {item?.message && <p>{item?.message}</p>}
-            </Alert>
-          );
-        })}
+      {alerts.map((item) => {
+        return (
```

Comment:
Here's another place where you can get rid of the block and the `return` statement, if you like.

Reply:
Updated!